src/components/InfoModal.tsx

```diff
@@ -55,36 +55,35 @@ const ResponsibleSelector = ({ employeeTask }: { employeeTask: IEmployeeTask })
 
   useMemo(() => employeeTask, [employeeTask, employeeTask?.responsible]);
 
-  const onSubmit = handleSubmit((formData) => {
+  const onSubmit = handleSubmit(async (formData) => {
     if (formData.responsible?.id !== employeeTask.responsible.id) {
-      axios
-        .all([
-          axios.put(`/api/employeeTasks/${employeeTask.id}`, {
-            completed: employeeTask.completed,
-            dueDate: employeeTask.dueDate,
-            responsibleId: formData.responsible?.id,
-          }),
-          axios.post('/api/notification', {
+      try {
+        await axios.put(`/api/employeeTasks/${employeeTask.id}`, {
+          completed: employeeTask.completed,
+          dueDate: employeeTask.dueDate,
+          responsibleId: formData.responsible?.id,
+        });
+        if (formData.responsible.employeeSettings.notificationSettings.includes('DELEGATE')) {
```

Comment:
En ide for litt mer beskrivende kode:
```suggestion
        const employeeWantsDelegateNotifications = formData.responsible.employeeSettings.notificationSettings.includes('DELEGATE');
        if (employeeWantsDelegateNotifications) {
```